astropy/modeling/functional_models.py

```diff
@@ -1711,40 +1843,54 @@ def _hum2zpf16c(z, s=10.0):
 
         # Optimized (single fraction) Humlicek region I rational approximation for n=16, delta=1.35
 
-        AA = np.array([+46236.3358828121,   -147726.58393079657j,
-                       -206562.80451354137,  281369.1590631087j,
-                       +183092.74968253175, -184787.96830696272j,
-                       -66155.39578477248,   57778.05827983565j,
-                       +11682.770904216826, -9442.402767960672j,
-                       -1052.8438624933142,  814.0996198624186j,
-                       +45.94499030751872,  -34.59751573708725j,
-                       -0.7616559377907136,  0.5641895835476449j])  # 1j/sqrt(pi) to the 12. digit
-
-        bb = np.array([+7918.06640624997, 0.0,
-                       -126689.0625,      0.0,
-                       +295607.8125,      0.0,
-                       -236486.25,        0.0,
-                       +84459.375,        0.0,
-                       -15015.0,          0.0,
-                       +1365.0,           0.0,
-                       -60.0,             0.0,
-                       +1.0])
+        # fmt: off
```

Comment:
Why is this still needed when it's reformatted anyway?

Reply:
Opps that should be a `# fmt: on`

Reply:
No this is right,  Black would expand this into a long single column of numbers. I actually prefer the parenthesis and brackets on their own lines as I have done here, but I retained the columns of the original.

Reply:
OK, yes, for 2 or 3 extra lines that's probably less distracting to the eye.